### PR TITLE
Support for Apache X-SendFile

### PR DESCRIPTION
**Pull request by saidul**
Hi I've added support for apache x-sendfile. Can you please review and merge this pull request?

### CODE REVIEW

File: Resources/config/services.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <parameters>
         <parameter key="igorw_file_serve.response_factory.php.class">Igorw\FileServeBundle\Response\PhpResponseFactory</parameter>
         <parameter key="igorw_file_serve.response_factory.sendfile.class">Igorw\FileServeBundle\Response\SendfileResponseFactory</parameter>
+		<parameter key="igorw_file_serve.response_factory.xsendfile.class">Igorw\FileServeBundle\Response\XsendfileResponseFactory</parameter>
```
Comment (stof) [inline Resources/config/services.xml]: wrong indentation (probably using tabs instead of spaces)

File: Response/XsendfileResponseFactory.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+use Symfony\Component\HttpFoundation\Response;
+
+class XsendfileResponseFactory extends AbstractResponseFactory
+{
+    protected function setResponseHeaders(Response $response)
+    {
+        parent::setResponseHeaders($response);
+
+        // $response->headers->set('X-Sendfile', realpath($this->fullFilename));
```
Comment (stof) [inline Response/XsendfileResponseFactory.php]: you should remove this commented code

File: Resources/config/services.xml (hunk 1 above)
Comment (saidul) [inline Resources/config/services.xml]: fixed

File: Response/XsendfileResponseFactory.php (hunk 2 above)
Comment (saidul) [inline Response/XsendfileResponseFactory.php]: done.

**Comment by igorw** [comment]
Thanks, merged!

**Pull request merged**